decidim-proposals/app/views/decidim/proposals/admin/proposals/show.html.erb

```diff
@@ -49,32 +49,34 @@
             </p>
           </div>
         </div>
-        <div class="card-section">
-          <div class="row column">
-            <span class="component__show-title"><%= t ".endorsers" %></span>
-            <ul id="proposal-endorsers-list">
-              <% presented_endorsers = endorsers_presenters_for(proposal).to_a %>
-              <% presented_endorsers.first(5).each do |presented_endorser| %>
-                <li>
-                  <%= link_to_if(
-                        presented_endorser.profile_path.present?,
-                        presented_endorser.name,
-                        presented_endorser.profile_path,
-                        target: :blank
-                      ) %>
-                </li>
-              <% end %>
-              <% if presented_endorsers.count > 5 %>
-                <li>
-                  <%= link_to(
-                        t(".n_more_endorsers", count: presented_endorsers.count - 5),
-                        resource_locator(proposal).path
-                      ) %>
-                </li>
-              <% end %>
-            </ul>
+        <% presented_endorsers = endorsers_presenters_for(proposal).to_a %>
```

Comment:
```suggestion
        <% presented_endorsers = endorsers_presenters_for(proposal).first(6).to_a %>
```

If there is a very long list of endorsers ( let's think tens or hundreds), this could generate a memory bloat.